benchmark/benchmark.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Would it be possible to use TypeScript for this and execute it with `ts-node`? Right now it's pretty simple but I expect that over time the benchmarks themselves may become more complex and could benefit from type checking, linting, etc.

Not sure how best to do that - could we possibly but the benchmarks embedded in the packages they test e.g. with a "/benchmark" subfolder like we have a "/test" subfolder? That also organizes benchmarks with each package in case some only apply e.g. to a particular exporter, etc.

Reply:
should we do it to `examples` as well ?

Reply:
Interesting question ... I actually think that giving users the ability to get to examples quickly feels valuable and so keeping it a top-level GitHub folder probably helps with that.

In contrast, a typical new user might not care about the details of benchmarks, though maybe if we advertised performance in some high performance level graphs in the README that could be cool. I'd mainly be interested in an instrumented vs. un-instrumented app performance there.

I can imagine us eventually wanting a top level "/tests" and "/benchmarks" folders as well that could test or benchmark a whole application that uses lots of different packages together, but for micro-benchmarks, I wonder if putting them with the package makes sense.

Reply:
+1 to instrumented vs non-instrumented. I think either benchmarking each plugin, or just providing a package which lets you easily DIY as you've suggested would be awesome.